news/4.34/jdt.html

```diff
@@ -134,6 +134,22 @@ <h2>Java Views and Dialogs</h2>
 			<img src="images/move-filtersbutton-to-top-other-way.png" alt="old menu">
 		  </p>
       </td>
+	  <tr id="experimental-preference">
```

Comment:
Please use a more specific ID, e.g.
```suggestion
	  <tr id="rescale-on-runtime-preference">
```